{"id":26705,"className":"javax.xml.ws.ProtocolException","methodName":"ProtocolException","signature":"\u003cjavax.xml.ws.ProtocolException: void ProtocolException(java.lang.Throwable)\u003e","javadoc":"/** \n * Constructs a new runtime exception with the specified cause and a  detail message of \u003ccode\u003e(cause\u003d\u003dnull ? null : cause.toString())\u003c/code\u003e  (which typically contains the class and detail message of  cause). This constructor is useful for runtime exceptions  that are little more than wrappers for other throwables.\n * @param cause the cause (which is saved for later retrieval by the\u003ccode\u003eThrowable.getCause()\u003c/code\u003e method). (A \u003ccode\u003enull\u003c/code\u003e value is  permitted, and indicates that the cause is nonexistent or  unknown.)\n */\n","tf":{"freq":{"littl":1,"protocol":1,"except":3,"detail":2,"save":1,"nonexist":1,"later":1,"messag":2,"permit":1,"runtim":2,"typic":1,"use":1,"unknown":1,"specifi":1,"caus":8,"indic":1,"wrapper":1,"retriev":1,"string":1,"contain":1,"valu":1,"method":1,"throwabl":2,"construct":1,"constructor":1},"maxFreq":8,"totalTerms":38},"loc":3}